lib/authentication/view/sign_up_view.dart

```diff
@@ -198,12 +139,10 @@ class _SignUpViewState extends State<SignUpView> {
         }
 
         fields[S.of(context).labelEmail] += S.of(context).stringEmailDomain;
-        nodes.asMap().forEach((i, node) {
-          if (i > 0) {
-            fields[filter.localizedLevelNames[i - 1]
-                [LocaleProvider.localeString]] = node.name;
-          }
-        });
+        if(dropdownController.path != null) {
+          fields.addAll(dropdownController.path);
```

Comment:
You could also write this if as one line with null-aware operators. Something like: `fields.addAll(dropdownController.path ?? {})`